src/api/l_graphics_pass.c

```diff
@@ -891,7 +891,11 @@ static int l_lovrPassCopy(lua_State* L) {
     uint32_t srcOffset = luax_optu32(L, 4, 0);
     uint32_t dstOffset = luax_optu32(L, 5, 0);
     const BufferInfo* info = lovrBufferGetInfo(buffer);
-    uint32_t limit = MIN(blob->size - srcOffset, info->length * info->stride - dstOffset);
+    uint32_t bufferSize = info->length * info->stride;
+    lovrCheck(srcOffset <= blob->size, "Source index is %d but blob is only %d bytes", srcOffset, (uint32_t) blob->size);
+    lovrCheck(dstOffset <= bufferSize, "Destination index is %d but buffer is only %d bytes", dstOffset, bufferSize);
```

Comment:
```suggestion
    lovrCheck(srcOffset <= blob->size, "Source byte offset is %d but Blob is only %d bytes", srcOffset, (uint32_t) blob->size);
    lovrCheck(dstOffset <= bufferSize, "Destination byte offset is %d but Buffer is only %d bytes", dstOffset, bufferSize);
```

Small suggestion to differentiate between index/offset terminology (index is 1-indexed and means "multiplied by byte stride" in context of buffers).

Reply:
Oh, I can commit this